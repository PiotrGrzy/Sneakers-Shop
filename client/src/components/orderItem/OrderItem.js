import React from 'react';

import './order-item.scss';

const OrderItem = ({ item }) => {
  const { brand, model, quantity, price } = item;
  return (
    <div className="order-item">
      <span className="order-item__brand">{brand}</span>
      <span className="order-item__model">{model}</span>
      <span className="order-item__quantity">{quantity}</span>
      <span className="order-item__price">{price}</span>
    </div>
  );
};

export default OrderItem;
