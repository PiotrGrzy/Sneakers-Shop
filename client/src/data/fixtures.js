export const item = {
  id: 'ck75wcleo155b0b843rlkt044',
  price: 129,
  gender: 'Men',
  category: ['Running'],
  imageMain: {
    id: 'ck75w55qoe0v90b201x2oqo8l',
    url: 'https://media.graphcms.com/JqjJAtRTlethzLQxhUtQ',
  },
  model: 'Zoom 210',
  imageSecondary: {
    id: 'ck75w55kfe0v40b209n4moxvp',
    url: 'https://media.graphcms.com/EVQOXidUTZksHn7HzJAQ',
  },
  sizes: '40 41 42 43 44 45 46 47',
  brand: 'Nike',
  trending: true,
};
