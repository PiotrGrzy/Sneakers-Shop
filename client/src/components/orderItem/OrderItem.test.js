import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });
import OrderItem from './OrderItem';
import { item } from '../../data/fixtures';

const props = {
  dataItem: { ...item, quantity: 1 },
};

describe('OrderItem', () => {
  const wrapper = shallow(<OrderItem item={props.dataItem} />);
  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
