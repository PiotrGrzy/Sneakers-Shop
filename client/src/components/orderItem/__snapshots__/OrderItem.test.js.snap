// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderItem renders properly 1`] = `ShallowWrapper {}`;
